packages/version/src/__tests__/__snapshots__/update-lockfile-version.spec.js.snap

```diff
@@ -11,7 +11,7 @@ Object {
     },
     "@my-workspace/package-2": Object {
       "requires": Object {
-        "@my-workspace/package-1": "^2.4.0",
+        "@my-workspace/package-1": "^2.3.4",
```

Comment:
this, and all other changes in the snapshot, should really stay at `2.4.0`, so there must be something wrong with your new code